API/Data/Metadata/ComicInfo.cs

```diff
@@ -54,6 +62,8 @@ public class ComicInfo
         public string StoryArc { get; set; } = string.Empty;
         public string SeriesGroup { get; set; } = string.Empty;
         public string AlternateNumber { get; set; } = string.Empty;
+        // This should really be -1 according to anansi-project spec
```

Comment:
Please remove these comments. I understand the spec, but there is no need to have -1 for this application. No one validates against the spec of the ComicInfo.